x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/dataframe/extractor/ExtractedFieldsDetectorTests.java

```diff
@@ -55,7 +55,7 @@ public void testDetect_GivenFloatField() {
             .addAggregatableField("some_float", "float").build();
 
         ExtractedFieldsDetector extractedFieldsDetector = new ExtractedFieldsDetector(
-            SOURCE_INDEX, buildOutlierDetectionConfig(), 100, fieldCapabilities, Collections.emptyMap());
+            buildOutlierDetectionConfig(), 100, fieldCapabilities, Collections.emptyMap());
```

Comment:
Is it now possible to remove `SOURCE_INDEX` constant? Or is it still used?

Reply:
still used when we create the job configs